zabbix/helper.go

```diff
@@ -0,0 +1,88 @@
+package provider
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"strings"
+	"time"
+
+	"github.com/claranet/go-zabbix-api"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func randStringNumber(length int) string {
```

Comment:
I don't believe this provides enough entropy to be reliable.

As this is only used for the `zabbix_template_link` virtual resource, the id does not need to be a 5 digit number:
- https://github.com/claranet/terraform-provider-zabbix/pull/7/files#diff-a123979e667d7e94c39402bf439a5d37R28
- https://github.com/claranet/terraform-provider-zabbix/pull/7/files#diff-a123979e667d7e94c39402bf439a5d37R81